plugins/system_upgrade.py

```diff
@@ -511,6 +511,24 @@ def run_reboot(self):
         reboot()
 
     def run_download(self):
+        #  downstream hack to narrow missing upgrade path with modules between dist versions
+        if self.opts.command not in ['offline-upgrade', 'offline-distrosync']:
+            if dnf.base.WITH_MODULES:
+                module_container = self.base._moduleContainer
+                enabled_module_names = set()
+                for module in module_container.getModulePackages():
+                    if module_container.isEnabled(module):
+                        enabled_module_names.add(module.getName())
+
+                for module_name in enabled_module_names:
+                    enabled_stream = module_container.getEnabledStream(module_name)
+                    default_stream = module_container.getDefaultStream(module_name)
+                    if not (enabled_stream and default_stream):
```

Comment:
Please could you add a comment with following information? - It is a hotfix to prevent multiple module operation for the same module => prevents from an error: ...